app/routes/index.js

```diff
@@ -62,9 +62,9 @@ module.exports = function(app)
 	app.use('/embed', require('./embed'));
 	app.use('/docs', access.isAuthenticated, require('./docs'));
 	app.use('/games/add', access.isEditor, require('./games/add'));
-	app.use('/games/search', access.isAdmin, require('./games/search'));
+	app.use('/games/search', access.isAuthenticated, require('./games/search'));
```

Comment:
Did I... start fixing this several days ago and then forget to finish it?

Reply:
I believe you did, I just didn't know where you put the fix, so I just fixed it as well while I was fixing the bug preventing readonly users from seeing the game page.

Reply:
Ahhh I know what happened. I fixed it on a fork from 1.5.4, but never made any changes to the main branch.